.github/workflows/ammr-doc.yaml

```diff
@@ -22,21 +22,22 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         
-      - uses: prefix-dev/setup-pixi@v0.4.3
+      - uses: prefix-dev/setup-pixi@v0.5.0
         with:
-          manifest-path: Docs/pixi.toml
           cache: true
```

Comment:
This only brings you something if you specify `environments: docs`. Otherwise it will cache the `default` environment (which is not used) and install the docs environment manually every time.